select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -115,66 +110,6 @@ public AbstractSelect2Choice(String id, IModel<M> model, ChoiceProvider<T> provi
 	}
 
 	/**
-	 * Construct.
-	 *
-	 * @param id
-	 *            markup id
-	 * @param model
-	 *            model for select
-	 * @param collection
-	 *            list options for select
-	 */
-	public AbstractSelect2Choice(String id, IModel<M> model, List<T> collection)
```

Comment:
First of all: you don't need a Wicket component for that, just create your (static) dropdown and use select2 directly. We could consider implementing a Select2Behavior for that.

Second: the way this is implemented forces the user to supply a static List of choices, else an exception is thrown during construction.

This was already discussed and accepted for master branch, just backporting it now as I'm currently using wicket-6.x in my $ project.